docs/reference/rest-api/common-parms.asciidoc

```diff
@@ -42,6 +42,7 @@ tag::allow-no-indices[]
 the request does *not* return an error
 if a wildcard expression
 or `_all` value retrieves only missing or closed indices.
+Defaults to `true`.
```

Comment:
I'm afraid that the default value here depends on the API, or is it `true` for all APIs?

Reply:
I tried some of them like HEAD, _mapping, _forgemerge and they behave the same.

Reply:
The default is API dependant. For example the put mapping api defaults to false.

Reply:
I see, should we just add a sentence that the default value depends on the API instead?

Reply:
You can add the default value where the definition is included instead:

```
include::{docdir}/rest-api/common-parms.asciidoc[tag=allow-no-indices]
+
Defaults to `true`.
```

Reply:
@jrodewig ++